.github/workflows/static-code-analysis.yml

```
@@ -26,14 +26,44 @@ jobs:
         run: composer i
 
       - name: Psalm
-        run: composer run psalm -- --monochrome --no-progress --output-format=github --update-baseline || ( git diff -- . ':!lib/composer' && exit 1 )
+        run: composer run psalm -- --monochrome --no-progress --output-format=github --update-baseline --report=results.sarif
 
-      - name: Check diff
+      - name: Show potential changes in Psalm baseline
```

Shall this be shown as well in case if error, like before?
```suggestion
      - name: Show potential changes in Psalm baseline
        if: always()
```

---

Maybe. Will psalm update the baseline even on failure?

---

Not sure, but there might be a reason why the full `git diff` was previously shown on Psalm failure only (and on success the baseline file diff only).

---

Added the `if: always()` in both jobs.